src/components/Experiment.vue

```diff
@@ -128,4 +189,49 @@ export default {
 .header .col:last-child {
   text-align: right;
 }
+
+/**
+ Default styles
+ */
+
+h1,
+h2,
+h3,
+h4 {
+  color: rgba(0, 0, 0, 0.87);
+  font-family: 'Playfair Display', 'Noto Serif', serif;
```

Comment:
Nice serif font choice :) Though we need to provide the font somehow, or you would expect the user to fallback on the default font if they don't have the Google fonts installed.

Reply:
I have no idea, I copied the styles from the old magpie.